gnomad_qc/v3/resources/annotations.py

```diff
@@ -36,7 +39,7 @@ def get_info(split: bool = True) -> VersionedTableResource:
     )
 
 
-def get_vqsr_filters(
+def get_filters(
```

Comment:
Just realized this ended up in a lot of PRs, I changed back to `get_vqsr_filters` in my other PRs